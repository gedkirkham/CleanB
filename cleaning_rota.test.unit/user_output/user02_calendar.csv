Date,Kitchen,Bedroom,
01/09/2018,,,
08/09/2018,Ged,Razvan,
15/09/2018,Razvan,,
22/09/2018,Ali,,
29/09/2018,Peter,,
06/10/2018,Ged,Ali,
13/10/2018,Razvan,,
20/10/2018,Ali,,
27/10/2018,Peter,,
03/11/2018,Ged,Peter,
10/11/2018,Razvan,,
17/11/2018,Ali,,
24/11/2018,Peter,,
01/12/2018,Ged,Ged,
08/12/2018,Razvan,,
15/12/2018,Ali,,
22/12/2018,Peter,,
