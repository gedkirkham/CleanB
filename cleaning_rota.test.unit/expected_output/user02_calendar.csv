Date,Kitchen,Bedroom
18/08/2018,,
25/08/2018,Ged,Razvan
01/09/2018,Razvan,
08/09/2018,Ali,
15/09/2018,Peter,
22/09/2018,Ged,Ali
29/09/2018,Razvan,
06/10/2018,Ali,
13/10/2018,Peter,
20/10/2018,Ged,Peter
27/10/2018,Razvan,
03/11/2018,Ali,
10/11/2018,Peter,
17/11/2018,Ged,Ged
24/11/2018,Razvan,
01/12/2018,Ali,
08/12/2018,Peter,
